src/ReactiveUI.Blend/FollowObservableStateBehavior.cs

```diff
@@ -5,32 +5,34 @@
 using System;
 using System.Reactive.Linq;
 
-#if !NETFX_CORE
+#if NETFX_CORE
```

Comment:
I also flipped a few of these around to match the other changes - happy to revert if you want to minimise change.